docs/api-reference/_category_.json

```diff
@@ -1,8 +1,4 @@
 {
-  "label": "Reference",
-  "position": 4,
-  "link": {
-    "type": "generated-index",
-    "description": "A complete reference for DBOS."
-  }
+  "label": "TypeScript Reference",
+  "position": 4
```

Comment:
I am sure there is a reason for the description to get dropped, but what is it?

Reply:
It makes navigation much smoother as now as you can click on the tabs to open/close them instead of clicking on the little arrow next to the tab.

Reply:
Demo apps could use the description page, it gives you a nice layout.